### PR TITLE
fixed _.minBy lodash version problem with the datatables

### PR DESCRIPTION
**Pull request by hackmod**
this PR fix some annoying bug with the dataTables plugin with the lodash `_.minBy`
 * update the latest datatables
 * fixup some datatables css styling

image

- `_.minBy` does not supported by the lodash v3.x
- we already use the lodash v4.x but the outdated dataTables.js make some trouble with the lodash.
- after loading address page the lodash version is downgraded unexpectedly by the dataTables.js plugin.

### CODE REVIEW

**Comment by btenterprise2020** [comment]
So this issue is already fixed on https://github.com/ethereumclassic/explorer/pull/146 right? 😄

**Comment by hackmod** [comment]
yes. PR 146 include datatables.js update and have no lodash.js related `_.minBy` issue.

**Comment by btenterprise2020** [comment]
Rebase your PR!

**Comment by btenterprise2020** [comment]
Just a small note to fix it later: `DevTools failed to parse SourceMap: https://classic.btenterprise.io/plugins/async.min.map`

File: public/plugins/datatables/datatables.all.min.js [hunk 1]
```diff
@@ -1,525 +1 @@
-/*
```
Comment (btenterprise2020) [inline public/plugins/datatables/datatables.all.min.js]: Is there any reason to delete comments?

File: public/js/main.js [hunk 2]
```diff
@@ -130,7 +130,7 @@ BlocksApp.config(['$stateProvider', '$urlRouterProvider', function($stateProvide
                         insertBefore: '#ng_load_plugins_before', // load the above css files before '#ng_load_plugins_before'
                         files: [
                              '/js/controllers/AddressController.js',
-                            '/plugins/datatables/datatables.min.css',
+                            //'/plugins/datatables/datatables.min.css',
```
Comment (btenterprise2020) [inline public/js/main.js]: Why did you excluded the datatables.min.css?

File: public/views/address.html [hunk 3]
```diff
@@ -62,7 +62,7 @@
           <div class="tab-content">
             <div class="tab-pane table-container" ng-class="{active: activeTab == 'tab_addr_1' || !activeTab}" id="tab_addr_1"><!-- Datatable tab -->
 
-              <table class="table table-striped table-hover" id="table_txs" style="table-layout:fixed;width: 98% !important;">
+              <table class="table table-striped table-bordered table-hover" id="table_txs" style="table-layout:fixed;width: 98% !important;">
```
Comment (btenterprise2020) [inline public/views/address.html]: What does this thing do?

**Comment by btenterprise2020** [comment]
Currently, I am working on to apply partial gulp support, it may help managing plugin versions.

**Pull request closed without merging**